systemtests/network_test.go

```diff
@@ -266,6 +266,51 @@ func (s *systemtestSuite) testNetworkAddDeleteNoGateway(c *C, encap string) {
 	}
 }
 
+func (s *systemtestSuite) TestNetworkEPGAddDelete(c *C) {
+
+	var err error
+	numContainers := 4
+	networkName := "net1"
+	epgName := "epg1"
+
+	for i := 0; i < s.iterations; i++ {
+		// Create Endpoints
+		network := &client.Network{
```

Comment:
I'll add support for multiple EPGs, but not sure if adding the multiple networks will help here and might just end up increasing test execution time. That part is already been tested in "TestNetworkAddDelete*" routines. Does adding/deleting EPG have a different effect in L3 mode? If so, I'll add support for that.